src/context/Categorie.jsx

```diff
@@ -0,0 +1,43 @@
+import {createContext, useEffect, useState} from 'react';
+import {usePermissions} from "../hooks/useUser";
+import {useCategorieList} from "../hooks/useCategorieList";
+
+const CategorieContext = createContext({});
+
+export const CategorieProvider = ({children}) => {
+    const [categorie, setCategorie] = useState("");
+    const [catColor, setCatColor] = useState("");
+
+    const permissions = usePermissions();
+    const usecategorielist = useCategorieList();
+
+    const majColor = (listCategorie,catName) => {
+        // console.log("MàJ de la couleur")
+        listCategorie.forEach((line) => {
+            if (line.nom === catName) {
+                setCatColor(line.color);
+            }
+        });
+    }
+
+    useEffect(() => {
+        if (Object.keys(permissions).length) {
+            // console.log("permission trouvé! affectation de la catégorie")
+            if (permissions.entities_manageable.length) {
```

Comment:
Lors de ma review, le champ entities_manageable n'étais pas renvoyé par le back (branche main).
La modification n'est pas obligatoire mais ne mange pas de pain : 
```suggestion
            if (permissions.entities_manageable?.length) {
```
Pour être certain de pouvoir lire length ou d'avoir un undefined propre (sans crash) si entities_manageable n'existe pas dans l'objet.